src/Form/Form.tsx

```diff
@@ -145,13 +145,13 @@ const Form: FormComponent = React.forwardRef((props: FormProps, ref: React.Ref<F
   const [_formValue, setFormValue] = useState(formDefaultValue);
   const [_formError, setFormError] = useState(formError || {});
 
```

Comment:
Here `useState(formDefaultValue)` can be replaced with `useControlled(formValue, formDefaultValue)`. This way you can get the ref through the third value returned by `useControlled` .

Need to pay attention to whether there is an impact on other functions.

https://github.com/rsuite/rsuite/blob/553241eb70f3a70c3867489e6d08ab1f4e41f1e7/src/utils/useControlled.ts#L21-L40